bb-pilcom/bb-pil-backend/src/verifier_builder.rs

```diff
@@ -182,13 +183,15 @@ impl VerifierBuilder for BBFiles {
         // Execute ZeroMorph rounds. See https://hackmd.io/dlf9xEwhTQyE3hiGbq4FsA?view for a complete description of the
         // unrolled protocol.
         // NOTE: temporarily disabled - facing integration issues
-        // auto pairing_points = ZeroMorph::verify(commitments.get_unshifted(),
+        // auto opening_claim = ZeroMorph::verify(commitments.get_unshifted(),
         //                                         commitments.get_to_be_shifted(),
         //                                         claimed_evaluations.get_unshifted(),
         //                                         claimed_evaluations.get_shifted(),
         //                                         multivariate_challenge,
+        //                                         pcs_verification_key->get_g1_identity(),
```

Comment:
This is used in the [eccvm verifier](https://github.com/AztecProtocol/aztec-packages/blob/c661fdebbd3de0b3f1cb09c1d0702eea81dc2666/barretenberg/cpp/src/barretenberg/eccvm/eccvm_verifier.cpp#L66) and the result is identical to the value of `Commitment::one()` used in some [other calls](https://github.com/AztecProtocol/aztec-packages/blob/c661fdebbd3de0b3f1cb09c1d0702eea81dc2666/barretenberg/cpp/src/barretenberg/ultra_honk/ultra_verifier.cpp#L83) of this function.

Not sure if there is any other difference, but my preference is for this one.

Reply:
Please if you are not sure, use rather Commitment::one() or ask a crypto_dev team member.